app/connectors/CalculatorConnector.scala

```diff
@@ -29,75 +30,87 @@ import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
 class CalculatorConnector @Inject()(val servicesConfig: ServicesConfig,
-                          val http: HttpClient)(implicit val ec: ExecutionContext) {
-
-  val serviceUrl: String = servicesConfig.baseUrl("capital-gains-calculator")
+                                    val http: HttpClientV2)(implicit val ec: ExecutionContext) {
+  val headers: Seq[(String, String)] = Seq("Accept" -> "application/vnd.hmrc.1.0+json")
 
   implicit val hc: HeaderCarrier = HeaderCarrier()
+  private val serviceUrl: String = servicesConfig.baseUrl("capital-gains-calculator")
 
-  val headers = Seq("Accept" -> "application/vnd.hmrc.1.0+json")
-
-  def getMinimumDate()(implicit hc : HeaderCarrier): Future[LocalDate] = {
-    http.GET[LocalDate](s"$serviceUrl/capital-gains-calculator/minimum-date", headers = headers)
+  def getMinimumDate()(implicit hc: HeaderCarrier): Future[LocalDate] = {
+    http.get(url"$serviceUrl/capital-gains-calculator/minimum-date")
+      .transform(_.addHttpHeaders("Accept" -> "application/vnd.hmrc.1.0+json"))
```

Comment:
Why can't we use `headers`  variable. The drawback with using the string value is incase it changes we'll have to update it everywhere. I'd prefer using `val headers` rather than `"Accept" -> "application/vnd.hmrc.1.0+json"` in `addHttpHeaders`

Reply:
Made changes according to the above comments.